### ISSUE TITLE
Pop 20.04 won't boot after install

### ISSUE THREAD

**Issue by markallanson** [pop-os/pop#1200]
**Distribution (run `cat /etc/os-release`):**
 Pop 20.04
 
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**
Kernel?

**Issue/Bug Description:**
After updating everything from Pop! Shop upon reboot I was taken directly to bios. Trying to boot manually into pop using boot selector just punts me right back to bios.

Tried fixing the bootloader per instructions and no success.

Tried fresh reinstalling (wiping all partitions) using both the normal and nvidia ISOs as I figured this was an nvidia driver issue but the problem happens with both.

Tried installing Ubuntu 20.04.1, Manjaro and Fedora 32 and all boot without any issues at all.

There are no diagnostics shown on screen, no errors, I am just sent straight to bios.

**Steps to reproduce (if you know):**
Install the current (as of writing) Pop!_OS iso onto a Lenovo Thinkpad P43s

**Expected behavior:**
It boots correctly into Pop.

**Other Notes:**
Lenovo P43s laptop - 48GB memory, Nvidia Quadro P520 GPU.

**Comment by markallanson** [comment]
If there is anywhere I might find some logs please let me know and I will go hunting using the live usb (which of course works fine)

**Comment by s3bc40** [comment]
Hey, I have the same problem. It's been four days since I am trying to solve this problem. I have just reinstalled Pop!_OS with my live USB boot, and it does not work after the update. It might come from the new kernel. 

I posted on reddit [r/pop_os](https://www.reddit.com/r/pop_os/comments/igya2z/boot_problem_with_pop_os_2004_lts_and_the_linux/), but I did not get any review, so I thought I was alone. 

**cat /etc/os-release**:
NAME="Pop!_OS"
VERSION="20.04 LTS"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 20.04 LTS"
VERSION_ID="20.04"
HOME_URL="https://pop.system76.com"
SUPPORT_URL="https://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal
LOGO=distributor-logo-pop-os

**Old-conf kernel before update :**
5.4.0-7625-generic 

**Bios info:**
System-manufacturer :  ASUSTeK COMPUTER INC.
System-product-name :  X453MA
Bios-release-date :  07/09/2014
Bios-version :  X453MA.209

I tried everything but I am lost right now. I am also in EFI mode.

Hope we find a solution to be able to use Pop!_OS

**Comment by mmstick** [comment]
@s3bc40 Do you have NVIDIA graphics?

@markallanson Did you try the Intel ISO?

**Comment by markallanson** [comment]
> @markallanson Did you try the Intel ISO?

Yep as mentioned in the big report above I have tried both the nvidia and intel ISO’s.

**Comment by s3bc40** [comment]
@mmstick No this is a Intel Atom® Processor Z3700 Series.

I took a screenshot of `neofetch`, posted on my reddit post on r/pop_os.

![xm6ubgvuecj51](https://user-images.githubusercontent.com/32032033/92449236-08ad2f00-f1ba-11ea-84a8-d023a38370e1.png)

Pop!_OS is running on the old kernel, but after the last update it can not boot on the last kernel.

**Comment by markallanson** [comment]
I thought i'd try again as i'd been avoiding doing any major work on my laptop since I wanted to get Pop running, and figured Pop 20.10 may have fixed it.

Unfortunately it did not - the symptoms are the same.

This time I played with a bunch of bios settings and nothing could get it to book.

Then I decided to try legacy boot mode instead of uefi and lo and behold after reinstalling pop for the 12th time in this saga it actually booted ok.

So - this is a problem with my thinkpad and uefi bios.

**Comment by markallanson** [comment]
@mmstick any chance you could add the 20.10 label please. maybe also tag it with uefi?

**Comment by lopopylopy** [comment]
i am also having the same issue. exact symptoms

**Comment by mamunozgonzalez** [comment]
Same issue here. Tried nvidia and intel versions from PopOS 20.10. Used Rufus and Etcher to create the bootable USB but in all cases, after burning it, it won't boot (also tried with different usb drives).

When the usb is plugged, a screen showing "Try or install Pop_OS" appears and after clicking on that button, it shows a black screen with the following message:

error: /casper_pop-os_20.10_amd64_nvidia_debug_17/vmlinuz.efi has invalid signature
error: you need to load the kernel first.

Press any key to continue...

**Comment by mmstick** [comment]
@mamunozgonzalez Did you disable secure boot?

**Comment by mamunozgonzalez** [comment]
Thanks a million @mmstick . I have disabled secure boot and works great!
Sorry for missing that, I tested first with ubuntu and forgot to disable it
